lumicks/pylake/nb_widgets/image_editor.py

```diff
@@ -0,0 +1,187 @@
+import numpy as np
+from dataclasses import dataclass, field
+import matplotlib.pyplot as plt
+from matplotlib.axes import Axes
+
+
+class ImageStackAxes(Axes):
+    def __init__(
+        self, *args, image=None, frame=0, channel="rgb", show_title=True, plot_kwargs={}, **kwargs
```

Comment:
unfortunately as far as I know all of the custom arguments need to be keyworded. I've added a check that `image is not None` and since this is an internal class I think it'll be ok

Reply:
We briefly discussed this already, but just to resolve this point, as long as there is an argument that collects positional arguments before it, like you have here, you don't need to default keyworded arguments.